src/commands.cc

```diff
@@ -2091,6 +2091,19 @@ static Completions complete_face(const Context& context, CompletionFlags flags,
                      transform([](auto& entry) -> const String& { return entry.key; }))};
 }
 
+static String face_doc_helper(const Context& context, CommandParameters params)
+{
+    if (params.size() < 2)
+        return "";
```

Comment:
```suggestion
        return {};
```

Creates an empty string without having to parse a const char*.